tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/locales/scriptToExemplars.txt

```diff
@@ -0,0 +1,18 @@
+#Contains sample aux exemplars, based on differences between exemplars of different scripts.
```

Comment:
What's the source of this data ?

Reply:
The data is CLDR itself. Determining all the letters that are used in other languages that have the same script, and doing some small adjustments to that.